spec.html

```diff
@@ -26296,7 +26295,7 @@
         </emu-alg>
       </emu-clause>
 
-      <!-- es6num="20.3.1.14" -->
+      <!-- es6num="20.3.1.13" -->
```

Comment:
These numbers should not be adjusted. They are historical backreferences to a previous version of the specification.